### DISCUSSION TITLE
You rock! Open and transparent code is the way towards a better web!

### DISCUSSION THREAD

**Discussion by sinful-mads** [mantinedev/mantine#2309] [Kind words]
I just wanted to thank you for having a transparent code base. We are using Radix UI ourselves and I needed help developing and implementing a method that could scroll to a specific element in the scrollarea component. I found your code and looked at your examples and source code and figured out how to do it in our project. Absolute class. Thank you very much :)

**Comment by rtivital** [comment]
Thanks for kind words @sinful-mads!
